mne/realtime/fieldtrip_client.py

```diff
@@ -333,11 +333,9 @@ def stop_receive_thread(self, stop_measurement=False):
         Parameters
         ----------
         stop_measurement : bool
-            Also stop the measurement.
+            unused, for compatibility.
```

Comment:
I don't understand the comment. Is this parameter to be deprecated?

Reply:
``RtEpochs`` expects all client implementations to have this parameter (https://github.com/mne-tools/mne-python/blob/master/mne/realtime/epochs.py#L312).  Since there is no "BaseClient" implementation/interface, this is unfortunately not visible in the type hierarchy. For example, the ``RtClient`` implementation supports stopping the actual measurement (https://github.com/mne-tools/mne-python/blob/master/mne/realtime/client.py#L299).
The ``FieldTripClient``, however, does not support stopping the measurement and this change in the doc should merely clarify the existing behavior.
Would you prefer reverting this change or elaborating on the reason for the existence of the parameter?

Reply:
@hekolk there is this [old pull request](https://github.com/mne-tools/mne-python/pull/4110/files) for implementing a `BaseClient`. Maybe something to resurrect if ever you're bored :)